### PR TITLE
clean up code

### PR DESCRIPTION
**Pull request by SgtRaven**
clean up code that is not being used

### CODE REVIEW

File: src/main/cpp/CommandMessenger.cpp [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #pragma once
 #include <string>
 
-class CommandMessenger {
```
Comment (KadenKreisler) [inline src/main/cpp/CommandMessenger.cpp:4]: This file could just be removed

File: src/main/cpp/RobotContainer.cpp [hunk 2]
```diff
@@ -194,130 +157,61 @@ void RobotContainer::ConfigureButtonBindings()
...
 frc2::CommandPtr RobotContainer::GetAutonomousCommand()
 {
   m_shooter.zeroIntergralVal();
 
   chosenAuto = m_chooser.GetSelected();
 
-
-  std::vector<frc::Pose2d> twoNotePoses{
+  std::vector<frc::Pose2d> twoNotePoses
+  {
```
Comment (KadenKreisler) [inline src/main/cpp/RobotContainer.cpp:207]: Some of these paths in the .cpp could probably be removed or moved to the .h

File: src/main/include/subsystems/ColorSensorSubsystem.h [hunk 3]
```diff
@@ -6,28 +6,21 @@
```
Comment (KadenKreisler) [inline src/main/include/subsystems/ColorSensorSubsystem.h:6]: This has been irrelevant for a while and should be removed

**Review by KadenKreisler — CHANGES_REQUESTED** [review]
you should remove colorSubsystem.cpp entirely

File: src/main/cpp/CommandMessenger.cpp (hunk 1 above)
Comment (SgtRaven) [reply src/main/cpp/CommandMessenger.cpp:4]: this has been resolved

File: src/main/cpp/RobotContainer.cpp (hunk 2 above)
Comment (SgtRaven) [reply src/main/cpp/RobotContainer.cpp:207]: we're going to resolve this later

File: src/main/include/subsystems/ColorSensorSubsystem.h (hunk 3 above)
Comment (SgtRaven) [reply src/main/include/subsystems/ColorSensorSubsystem.h:6]: this has been removed

**Pull request merged**